include/deal.II/lac/cuda_vector.h

```diff
@@ -110,10 +109,10 @@ namespace LinearAlgebra
        * <tt>swap(u,v)</tt> that simply calls <tt>u.swap(v)</tt>, again in
        * analogy to standard functions.
        *
-       * This function is virtual in order to allow for derived classes to
+       * This function is  in order to allow for derived classes to
        * handle memory separately.
```

Comment:
These lines should probably just be removed.

Reply:
Good point. I had removed all occurrences of `virtual` in this file because the class is no longer derived from anything. I didn't notice this place.